internal/controller/telemetry/otel_conf_gen_test.go

```diff
@@ -134,9 +132,16 @@ func TestOtelColConfComplex(t *testing.T) {
 	}
 	inputCfg.SubscriptionOutputMap = subscriptionOutputMap
 
-	inputCfg.TenantSubscriptionMap = map[v1alpha1.NamespacedName][]v1alpha1.NamespacedName{}
+	inputCfg.TenantSubscriptionMap = map[string][]v1alpha1.NamespacedName{}
 	tenant := inputCfg.Tenants[0]
-	inputCfg.TenantSubscriptionMap[tenant.NamespacedName()] = getSubscriptionNamesFromSubscription(inputCfg.Subscriptions)
+
+	keysOfMap := func(subsMap map[v1alpha1.NamespacedName]v1alpha1.Subscription) (names []v1alpha1.NamespacedName) {
+		for key, _ := range subsMap {
+			names = append(names, key)
+		}
+		return
+	}
+	inputCfg.TenantSubscriptionMap[tenant.Name] = keysOfMap(inputCfg.Subscriptions)
```

Comment:
[nit]
We could use `maps.Keys` here instead:
https://pkg.go.dev/golang.org/x/exp/maps#Keys

Reply:
Checked @siliconbrain's [seqs](https://github.com/siliconbrain/go-mapseqs) library, but sadly seqs can't solve everything. :(

Reply:
there's always [mapseqs](https://github.com/siliconbrain/go-mapseqs) 😁